lib/ui/widgets/itinerary/itinerary_form.dart

```diff
@@ -55,7 +55,6 @@ class _ItineraryFormState extends State<ItineraryForm>
           nResults: 12,
           time: formData.datetime,
           arriveBy: formData.arriveBy,
-          allowBikeRental: _allowBikeRental,
```

Comment:
Please remove the definition of `_allowBikeRental`, it isn't used anywhere else.